app/keepers.go

```diff
@@ -642,6 +635,28 @@ func (app *NibiruApp) InitModuleManager(
 	app.configurator = module.NewConfigurator(
 		app.appCodec, app.MsgServiceRouter(), app.GRPCQueryRouter())
 	app.mm.RegisterServices(app.configurator)
+
+	// see https://github.com/cosmos/cosmos-sdk/blob/666c345ad23ddda9523cc5cd1b71187d91c26f34/simapp/upgrades.go#L35-L57
+	for _, subspace := range app.paramsKeeper.GetSubspaces() {
+		subspace := subspace
```

Comment:
Not needed if is not concurrent

Reply:
Makes sense. I copied that code from the SDK but you're right.